src/Course/List.hs

```diff
@@ -284,8 +287,11 @@ find =
 lengthGT4 ::
   List a
   -> Bool
-lengthGT4 =
-  error "todo: Course.List#lengthGT4"
+lengthGT4 Nil = False
+lengthGT4 (_ :. Nil) = False
+lengthGT4 (_ :. _ :. Nil) = False
+lengthGT4 (_ :. _ :. _ :. Nil) = False
+lengthGT4 _ = True
```

Comment:
we have defined `length` already... why not use it?

`lengthGT4 = (> 4) . length`

or 

`lengthGT4 xs = length xs > 4`